Tensile/Configs/rocblas_hgemm_hip_lite.yaml

```diff
@@ -275,7 +275,8 @@ BenchmarkProblems:
 
 LibraryLogic:
 #   ScheduleName: "vega10"
-    DeviceNames: ["Device 6863", "Device 6862", "Device 687f", "Device 6860", "Device 6861", "Vega 10 XTX [Radeon Vega Frontier Edition]", "Vega [Radeon RX Vega]"]
+    DeviceNames: ["Device 6863", "Device 6862", "Device 687f", "Device 6860", "Device 6861", "Vega 10 XTX [Radeon Vega Frontier Edition]", "Vega [Radeon RX Vega]",
```

Comment:
The original file has a bug: line 278 should be commented out.  The additional line 279 you added would have to commented out also.  Thanks.